assets/src/scripts/accessibility.js

```diff
@@ -7,9 +7,14 @@
  * @param {Boolean} isInteractive
  */
 function addSVGAccessibility(svg, {title, description, isInteractive}) {
-    svg.attr('title', title)
-        .attr('desc', description)
-        .attr('aria-labelledby', 'title desc');
+    svg.select('title').remove();
+    svg.select('desc').remove();
```

Comment:
fyi, if you haven't seen it, selections support commas: `svg.select('title, desc').remove()`

Reply:
👍